modules/ROOT/pages/oauth-grant-types-about.adoc

```diff
@@ -3,14 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-OAuth 2.0 specify the following methods (ie Grant Types) for requesting for a token.
+OAuth 2.0 provides the following grant types methods for requesting for a token:
 
 * AUTHORIZATION_CODE
 * IMPLICIT
 * RESOURCE_OWNER_PASSWORD_CREDENTIALS
 * CLIENT_CREDENTIALS
 
-For those RAML-based APIs in which the RAML should be updated with the OAuth 2.0 security schema, the following table maps the RAML grant types to the grant type names in the OAuth 2.0 policy configuration.
+For the RAML-based APIs in which the RAML must be updated with the OAuth 2.0 security schema, the following table maps the RAML grant types to the grant type names in the OAuth 2.0 policy configuration:
```

Comment:
typically there wouldn't be an article here at all, just "For RAML-based APIs in which the RAML..."
What does the reader have to do? Replace the RAML grant type with the corresponding OAuth 2.0 grant type? It's great to have the mapping, but I'm not sure what the task is (and therefore, why the info is here). if there's any way to break up this long sentence into two smaller ones, that'd be great. I couldn't try as I don't know what we're telling the reader to do here.

Reply:
Done! Thanks!